### PR TITLE
Reinstate nightly refreshes for schedule E data

### LINKED ISSUE #2813: Reinstate nightly updates for schedule E data
The recent work with the migrations did have one side effect:  we lost the nightly updates to schedule E.  That last bit of work around this had us removing all queue processing for schedule E and just [rebuilding the table each night](https://github.com/18F/openFEC/blob/master/webservices/tasks/refresh.py#L22) with the [table creation setup](https://github.com/18F/openFEC/blob/master/data/sql_setup/prepare_schedule_e.sql).

The `update_itemized` method was removed, though, and the table creation slightly modified with the migration work.  We'll have to devise a new way to make this refresh possible again.

### PR DESCRIPTION
**Pull request by ccostino**
Fixes #2813

This changeset adds support for schedule E nightly refreshes back into place.  The recent migration work accidentally removed the nightly refresh support for schedule E.

A huge h/t to @vrajmohan and @LindsayYoung for helping test and uncover this issue!

### CODE REVIEW

File: data/refresh/rebuild_schedule_e.sql [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+insert into ofec_sched_e_tmp
+select *,
+    cast(null as timestamp) as timestamp,
+    image_pdf_url(image_num) as pdf_url,
+    coalesce(rpt_tp, '') in ('24', '48') as is_notice,
+    to_tsvector(pye_nm) as payee_name_text,
+    now() as pg_date
+from fec_sched_e_notice_vw;
+
+update ofec_sched_e_tmp
```
Comment (vrajmohan) [inline data/refresh/rebuild_schedule_e.sql]: Why is this a separate step? Why not deal with it in the 2 `INSERTS` above?

File: data/refresh/rebuild_schedule_e.sql [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+-- Analyze the table
+analyze ofec_sched_e_tmp;
+
+-- Replace the existing table
+drop table if exists ofec_sched_e;
+alter table ofec_sched_e_tmp rename to ofec_sched_e;
+
+-- Rename indexes
+alter index ofec_sched_e_sub_id_pkey_tmp rename to ofec_sched_e_sub_id_pkey;
```
Comment (vrajmohan) [inline data/refresh/rebuild_schedule_e.sql]: All the `ALTER INDEXES` could be handled more succinctly with the function `rename_indexes`.

**Comment by vrajmohan** [comment]
I have no context to review the logic in this code. All I can do is comment on general soundness of the SQL in a domain-ignorant way. 😄

**Comment by vrajmohan** [comment]
Are there no tests for this code?

**Comment by LindsayYoung** [comment]
I am going to merge this as-is with the follow up task here to simplify the update process here

https://github.com/18F/openFEC/issues/2825

**Comment by ccostino** [comment]
@vrajmohan:  D'oh!  I thought the existing tests covered this case but perhaps not.  FWIW this is the same exact setup we had before prior to the DB migration, but @LindsayYoung and I were also talking more this morning about restructuring this as a materialized view.  We can be sure to get the appropriate tests in with that upcoming work.

**Pull request merged**